indico/modules/events/contributions/controllers/management.py

```diff
@@ -182,7 +183,9 @@ def _process(self):
         contrib_form_class = make_contribution_form(self.event_new)
         form = contrib_form_class(obj=FormDefaults(location_data=inherited_location), event=self.event_new)
         if form.validate_on_submit():
-            contrib = create_contribution(self.event_new, *_get_field_values(form.data))
+            contrib = Contribution()
```

Comment:
ok, now i see. you need a contribution from which you get the empty speakers list. it's a bit ugly. maybe add a comment so it's clear why you need this?

    contrib = Contribution()  # so we have an empty initial person list in flash_if_unregistered